### ISSUE TITLE
Update for TYPO3 13.4 LTS

### ISSUE THREAD

**Issue by Epoche-Napoleon** [jweiland-net/maps2#323]
Is an update to TYPO3 13.4 LTS planned? If so, is there already a time frame for this?

**Comment by terminal8-nz** [comment]
Any news on this issue?

**Comment by sfroemkenjw** [comment]
Yes, it is planned and current work can be get/tested from typo3_13_compatibility branch

**Comment by Epoche-Napoleon** [comment]
Thanks for update this Extension.

**Comment by sfroemkenjw** [comment]
Hi all,

I've released maps2 in version 12.0.0 for TYPO3 13 in TER and Composer/Packagist. Sorry, no TYPO3 12 compatibility.

Stefan
